File: prograimg_python_basic/Exercises/conditional_statements/conditional_statements_-_exercise/bonus_score.py
bonus = float()
points = int(input())

if points <= 100:
    bonus = 5

elif points > 1000:
    bonus = points * 0.10

else:
    bonus = points * 0.20

if points % 2 == 0:
    bonus = bonus + 1

elif points % 10 == 5:
    bonus = bonus + 2

print(bonus)
print(points + bonus)

File: JS/JavaScript/JS-Basics/ConditionalStatementsExercise/BonusScore.js
function bonus(inputs) {
    let number = Number(inputs[0]);

    let points = 0;

    if (number <= 100) {
        points = 5;
    }
    else if (number > 100 && number <= 1000) {
        points = number * 0.20;
    }
    else if (number > 1000) {
        points = number * 0.10;
    };

    if (number % 2 == 0) {
        points += 1;
    };

    if (String(number).slice(-1) == 5) {
        points += 2;
    };
    
    console.log(points);
    console.log(number + points)

}

bonus(["15875"])